self-tests.yml

```diff
@@ -544,6 +544,25 @@ tasks:
           target: revendor
       - func: run-make
         vars: { target: "test-repotracker" }
+  - name: dep-test
```

Comment:
Is this something we want to commit?

Reply:
I personally find it useful but if you think it's unneeded I'm happy to get rid of it.

Reply:
Is there a way to re-write this as a unit test? That's more how we should be testing functionality, as opposed to running an empty test.

Reply:
Otherwise I would just remove it now that your testing is done.

Reply:
I wanted more for an end-to-end staging test for functionality that involves dependencies, so I'll just get rid of it and stick it in my notebook in case this is needed again.